source/controller.cpp

```diff
@@ -81,6 +86,8 @@ void GZ_readController() {
         sCursorEnableDelay = 0;
         // GZCmd_processInputs();
     }
+
+    GZ_readZombieHoverInputs();

```

Comment:
I'm not a big fan of how this function always gets called regardless of whether or not the user has the zombie hover tool open.  I didn't see an easy way to check if the tool was open from this file, but maybe I'll figure it out tomorrow (or maybe this doesn't matter).